src/corner_kick/src/App.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * Specifies the root component of the application
+ */
+
+import * as React from 'react';
+import { Provider } from 'react-redux';
+
+import { Console } from './components/portals/Console';
+import { Main } from './components/portals/Main';
+import { Sidebar } from './components/portals/Sidebar';
+import { SidebarTitle } from './components/portals/SidebarTitle';
+import { Theme } from './containers/mainUI/Theme';
+import { createStore } from './store';
+
+const store = createStore();
+
+// Top-level React component in the application
+export const App = () => (
```

Comment:
I know this should be known and understood by anyone with a web dev background, but could we add a comment here saying something to the effect of `// This is *NOT* HTML code, it is ...`? Just to make it more clear for the likely case where people with minimal (at best) web experience come bumbling through here.

Reply:
Done
```
// Top-level React component in the application
// !!!! THIS ... IS ... NOT ... HTML !!!!
// IF YOU DIDN'T REALIZE THIS, PLEASE GO AND READ
// https://reactjs.org/docs/introducing-jsx.html
```